src/history/InferredQuorumUtils.cpp

```diff
@@ -18,52 +18,55 @@ namespace stellar
 {
 
 void
-checkQuorumIntersection(Config cfg)
+checkQuorumIntersection(Config const& cfg)
```

Comment:
I'm not sure I see the point of this change. Instead of doing pass-by-value we just are doing pass-by-const-reference then copy. I don't see how this is better, we haven't removed the copy.

Reply:
yes: it's functionally equivalent as there is still a copy.
The difference is that we don't have to keep reviewing large pass-by-value calls in the code base as we have explicit copies (either during code review or like this one when we have a warning from static analyzers).